tsconfig.json

```diff
@@ -1,6 +1,13 @@
 {
-	"extends": "./node_modules/@salesforce/dev-config/tsconfig",
 	"compilerOptions": {
+		"module": "commonjs",
+		"target": "es2021",
+		"lib": ["es2021"],
+		"sourceMap": true,
+		"declaration": true,
+		"moduleResolution": "node16",
```

Comment:
The old "node" moduleResolution inside of the @salesforce/dev-config doesn't work with the new estlint modules. See https://github.com/typescript-eslint/typescript-eslint/issues/7284

So I believe we need to use node16 at least.